samples/msal-angular-v3-samples/jest.config.js

```diff
@@ -0,0 +1,12 @@
+const fs = require("fs");
+const path = require("path");
+
+const sampleFolders = fs.readdirSync(__dirname, { withFileTypes: true }).filter(function(file) {
```

Comment:
Can we follow the same pattern here in case if someone wants to run just one sample/test (or if we need it in CI)?

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/176635ec210fa0eed51a4b774dcd42d972816498/samples/msal-browser-samples/VanillaJSTestApp2.0/jest.config.js#L7

Reply:
Thanks! Updated